examples/motoko_examples/threshold_ecdsa/canisters/azle/index.did

```diff
@@ -0,0 +1,90 @@
+type CreateCanisterArgs = record {
```

Comment:
This index.did seems awfully long to me. Any idea why?

Reply:
This has been addressed during the Rust rewrite